V2_MIGRATION_GUIDE.md

```diff
@@ -60,164 +63,129 @@ Additionally, any classes that implemented `ParameterizableRequest` or `AuthRequ
 - The `com.auth0.android.util.Telemetry` class has been renamed to `com.auth0.android.util.Auth0UserAgent`.
-- The `com.auth0.android.request.AuthorizableRequest` class has been removed. You can achieve the same result using the method in Request: `Request#addHeader("Authorization", "Bearer TOKEN_VALUE")`.
+- The `com.auth0.android.request.AuthorizableRequest` class has been removed. You can achieve the same result using the `addHeader("Authorization", "Bearer TOKEN_VALUE")`.
 - The `com.auth0.android.authentication.request.TokenRequest` class has been removed. The ability to set a Code Verifier, and any request headers and parameters has been moved to the `com.auth0.android.request.Request` interface.
 - The `com.auth0.android.authentication.request.DatabaseConnectionRequest` class has been removed. The ability to set any request headers and parameters has been moved to the `com.auth0.android.request.Request` interface.
 
-### Class changes
+### Constants removed
+
+- `ParameterBuilder.GRANT_TYPE_JWT` has been removed.
+- `ParameterBuilder.ID_TOKEN_KEY` has been removed.
+- `ParameterBuilder.DEVICE_KEY` has been removed.
+- `ParameterBuilder.ACCESS_TOKEN_KEY` has been removed.
+- `ResponseType.CODE`, `ResponseType.ID_TOKEN`, and `ResponseType.ACCESS_TOKEN` have been removed.
+
+### Class and Interface changes
 
 - `SignupRequest` now implements `AuthenticationRequest` instead of the now-removed `AuthRequest`
 - `AuthorizableRequest` now extends `Request` instead of the now-removed `ParameterizableRequest`
 - `BaseCallback` has been deprecated; use `Callback` instead.
 
-### Constructors removed or changed
+### Constructor changes
 
-#### AuthenticationAPIClient
+- `AuthenticationAPIClient` can no longer be constructed from a `Context`. Use Use `AuthenticationAPIClient(client: Auth0)` instead. You can create an instance of `Auth0` using a `Context`.
+- `UsersAPIClient` can no longer be constructed from a `Context`. Use UsersAPIClient(client: Auth0, token: String)` instead. You can create an instance of `Auth0` using a `Context`.
+- `SignupRequest` now requires the second parameter to be an `AuthenticationRequest`.
+- `ProfileRequest` now requires an `AuthenticationRequest` and a `Request<UserProfile, AuthenticationException>`.
 
-- `public AuthenticationAPIClient(@NonNull Context context)`. Use `public AuthenticationAPIClient(new Auth0(context))`.
+### Methods removed or changed
 
-#### UsersAPIClient
+#### `Auth0` removals and changes
 
-- `public UsersAPIClient(@NonNull Context context, @NonNull String token)`. Use `public UsersAPIClient(new Auth0(context), token)`.
+- `setOIDCConformant(boolean enabled)` has been removed. The SDK now only supports OIDC-Conformant applications.
+- `isOIDCConformant()` has been removed. The SDK now only supports OIDC-Conformant applications.
```

Comment:
can probably be merged with the line above, as they are getters/setters for the same property